products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -64,7 +64,7 @@ private class ProcessDelegateWrapper {
             context: Unmanaged.passUnretained(self).toOpaque(),
             process_started: { BuildSystem.toProcessDelegateWrapper($0!).delegate.processStarted(pid: $1) },
             process_had_error: { BuildSystem.toProcessDelegateWrapper($0!).delegate.processHadError(error: stringFromData($1!.pointee)) },
```

Comment:
Can you do the same thing for standard error?

Reply:
This API isn't for stderr, it's for llbuild to report internal errors which should be valid strings. I'd rather keep this consistent with `commandProcessHadError`

Reply:
Oh, I see. I briefly passed over this and assumed they were stderr/stdout counterparts due to the similar names.